### PR TITLE
Nopatch groff CVE-2000-0803

### PR DESCRIPTION
**Pull request by dmcilvaney**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
`groff` CVE-2000-0803 is unpatched. Centos & Fedora do not appear to address this CVE either.

###### Change Log  

- Nopatch `groff` CVE-2000-0803
- Unify changelog format for `groff.spec`

###### Does this affect the toolchain?  

NO

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2000-0803

###### Test Methodology

- make 'input-srpms' to validate spec file format.

### CODE REVIEW

**Comment by lpereira** [comment]
Very few people will use groff by itself these days, so the impact in Mariner, even if this is unpatched, is minimal.  (Groff is used by man pages, but that's fine as far as this vulnerability is concerned, because those are provided by the system.)

File: SPECS/groff/groff.spec [hunk 1]
```diff
@@ -42,15 +46,17 @@ rm -rf %{buildroot}%{_infodir}
 %{_defaultdocdir}/%{name}-%{version}/*
 %{_datarootdir}/%{name}/*
 %{_mandir}/*/*
-%changelog
-* Sat May 09 00:21:28 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.22.3-4
-- Added %%license line automatically
 
+%changelog
+*   Mon Sep 28 2020 Daniel McIlvaney <damcilva@microsoft.com> - 1.22.3-5
```
Comment (MateuszMalisz) [inline SPECS/groff/groff.spec:51]: There shouldn't be `-` before the version. (Same applies to Nick's entry).

**Review by MateuszMalisz — APPROVED** [review]

File: SPECS/groff/groff.spec (hunk 1 above)
Comment (dmcilvaney) [reply SPECS/groff/groff.spec]: Fixed

Comment (dmcilvaney) [reply SPECS/groff/groff.spec]: Fixed

**Pull request merged**